### PR TITLE
added HTV, MX player pro, MX player

### PR DESCRIPTION
**Pull request by Ratulhasan14789**

### CODE REVIEW

**Comment by Ratulhasan14789** [comment]
ops wrote in older version

File: README.md [hunk 1]
```diff
@@ -35,10 +35,10 @@ There is no `com.android.vpndialogs` in WSA. However, it's possible to manually
 appops set [package name] ACTIVATE_VPN allow
 ```
 
-#### Launching Android Apps with URI scheme
+#### Launching Android Apps with URL scheme
```
Comment (pratyakshm) [inline README.md:38]:
```suggestion
#### Launching Android Apps with URI scheme
```
?

File: README.md [hunk 2]
```diff
@@ -35,10 +35,10 @@ There is no `com.android.vpndialogs` in WSA. However, it's possible to manually
...
 Microsoft provides a custom URI scheme for WSA, making it easier to launch apps:
 ```shell
-wsa://[App Package Name]
+wsl://[App Package Name]
```
Comment (pratyakshm) [inline README.md:41]:
```suggestion
wsa://[App Package Name]
```

File: README.md [hunk 3]
```diff
@@ -173,7 +175,7 @@ wsa://com.android.settings
 | Office lens | 16.0.14527.20178 | ❌ || Might require GMS, cannot sign in |
 | Opera Browser Beta | 65.1.3381.61349 (x86_64) | ✔️ || Change app layout to Tablet Mode for a better experience |
 | Opera GX : Gaming Browser | 1.3.6 | ✔️ |||
-| Opera Mini Beta | 61.0.2254.59921 | 🆖 | Dark mode doesn't work due to inability to draw over apps | Crashes without GSM (with no warning) |
+| Opera Mini Beta | 61.0.2254.59921 | 🆖 || Crashes without GSM (with no warning) |
```
Comment (pratyakshm) [inline README.md:178]:
```suggestion
| Opera Mini Beta | 61.0.2254.59921 | ❌ | App crashes ||
```

**Review by pratyakshm — CHANGES_REQUESTED** [review]
All changes in this repo are reviewed then merged. You don't have to fix something that's not broken.

File: README.md [hunk 4]
```diff
@@ -162,6 +162,8 @@ wsa://com.android.settings
 | Monument Valley 2 | 2.0.3 | ✔️ |||
 | MT File Manager | 2.10.0 | ✔️ |||
 | Muslim Pro | 1.2.3 | 🆖 | Requires GMS |
+| MX player | 1.40.9 | ✔️ | Playstore says unsupported | Works fine after downloading from website |
+| MX player Pro | 1.39.13 | ⚠️ | App home crash at start | Can play video if opened from file manager |
```
Comment (pratyakshm) [inline README.md:166]:
```suggestion
| MX Player | 1.40.9 | ✔️ | | |
| MX Player Pro | 1.39.13 | ⚠️ | App crashes, but videos can be played from external sources ||
```

**Review by pratyakshm — CHANGES_REQUESTED** [review]

**Pull request merged**